src/libraries/System.Net.Http/ref/System.Net.Http.cs

```diff
@@ -299,6 +301,23 @@ public sealed class HttpRequestOptions : System.Collections.Generic.IDictionary<
         public void Set<TValue>(HttpRequestOptionsKey<TValue> key, TValue value) { throw null; }
     }
 
+    [SupportedOSPlatform("windows")]
+    [SupportedOSPlatform("linux")]
+    [SupportedOSPlatform("macos")]
+    [UnsupportedOSPlatform("browser")]
+
+    public sealed class Http3WebtransportSession : System.IAsyncDisposable, IDisposable
+    {
+        public long Id { get { throw null; } }
+        internal Http3WebtransportSession() { }
+        public static System.Threading.Tasks.ValueTask<Http3WebtransportSession?> ConnectAsync(Uri uri, HttpMessageInvoker invoker, System.Threading.CancellationToken cancellationToken) { throw null; }
```

Comment:
Why are this and below methods returning nullable objects?

Reply:
The `GetIncomingWebtransportStreamFromServerAsync` needs to be nullable in case of the stopping of the cancellation token so I can return null. The others can be non-nullable. Will implement this

Reply:
When cancellation token fires, it should throw OperationCanceledException.